docs/guide/use_cases/nlb_tls_termination.md

```diff
@@ -0,0 +1,129 @@
+
+## Configure
+
+### Generate TLS Certificate
+
+Create a public TLS certificate for the domain using AWS Certificate Manager (ACM). This is streamlined when the domain is managed by Route 53. Review the [AWS Certificate Manager Docs.](https://docs.aws.amazon.com/acm/latest/userguide/gs-acm-request-public.html#request-public-console)
+
+The domain name on the TLS certificate must correspond to the planned domain name for the kubernetes service. The domain name may be specified explicitly (e.g., tls-demo.gcline.us), or a wildcard certificate can be used (e.g., *.gcline.us).
+
+If the domain is registered with Route53, the TLS certificate request will automatically be approved. Otherwise, follow ACM console the instructions to create a DNS record to validate the domain. 
+
+After validation, the certificate will be available for use in your AWS account. 
+
+Note the ARN of the certificate, which uniquely identifies it in kubernetes config files. 
+
+![screenshot indicating location of ARN value in web console](cert_arn.png)
+
+### Create Service with new NLB
+
+Add annotations to a load balancer service to enable NLB TLS termination, before the traffic reaches Envoy. The annotations are actioned by the load balancer controller. [Review all the NLB annotations on GitHub.](https://kubernetes-sigs.github.io/aws-load-balancer-controller/latest/guide/service/annotations/)
+
+| annotation name | value | meaning | 
+| ----- | --- | ----- |
+| service.beta.kubernetes.io/aws-load-balancer-type | external | explicitly requires an NLB, instead of an ALB |
+| service.beta.kubernetes.io/aws-load-balancer-nlb-target-type | ip | route traffic directly to the pod IP |
+| service.beta.kubernetes.io/aws-load-balancer-scheme | internet-facing | An internet-facing load balancer has a publicly resolvable DNS name |
+| service.beta.kubernetes.io/aws-load-balancer-ssl-cert | "arn:aws:acm:..." | identifies the TLS certificate used by the NLB |
+| service.beta.kubernetes.io/aws-load-balancer-ssl-ports | 443 | determines the port the NLB should listen for TLS traffic on| 
+
+Example: 
+
+```
+apiVersion: v1
+kind: Service
+metadata:
+  name: MyAppSvc
+  namespace: dev
+  annotations:
+    service.beta.kubernetes.io/aws-load-balancer-type: external
+    service.beta.kubernetes.io/aws-load-balancer-nlb-target-type: ip
+    service.beta.kubernetes.io/aws-load-balancer-scheme: internet-facing
+    service.beta.kubernetes.io/aws-load-balancer-ssl-cert: "arn:aws:acm:us-east-2:185309785115:certificate/7610ed7d-5a81-4ea2-a18a-7ba1606cca3e"
+    service.beta.kubernetes.io/aws-load-balancer-ssl-ports: "443"
+spec:
+  externalTrafficPolicy: Local
+  ports:
+  - port: 443
+    targetPort: 80
+    name: http
+    protocol: TCP
+  selector:
+    app: MyApp
+  type: LoadBalancer
+```
+
+!!! warning
```

Comment:
TLS configuration should not cause duplicate NLB resources. Did you encounter any issues modifying existing service resources?

Reply:
Yes. I tried adding TLS termination to an existing NLB. It didn't show the cert in the NLB console until I deleted the NLB, and the controller recreated it.

Reply:
https://github.com/kubernetes-sigs/aws-load-balancer-controller/issues/1870